src/seedu/addressbook/ui/TextUi.java

```diff
@@ -151,22 +103,9 @@ private void showToUserAsIndexedList(List<String> list) {
 
     /** Formats a list of strings as a viewable indexed list. */
     private static String getIndexedListForViewing(List<String> listItems) {
-        final StringBuilder formatted = new StringBuilder();
-        int displayIndex = 0 + DISPLAYED_INDEX_OFFSET;
-        for (String listItem : listItems) {
-            formatted.append(getIndexedListItem(displayIndex, listItem)).append("\n");
-            displayIndex++;
-        }
-        return formatted.toString();
+        return formatter.getFormattedIndexedListForViewing(listItems);
```

Comment:
Can this line be replaced with `showToUser(Formatter.getFormattedIndexedListForViewing(list))` in the `showToUserAsIndexedList` method instead of creating a single line method?

Reply:
o yea! I didn't realize! thanks 👍